billing.rst

```diff
@@ -0,0 +1,146 @@
+\           **Total before taxes**                                                  €52.79
+=========== =================================================== ======== ========== ======
+
+Am I gonna be charged after trial?
+----------------------------------
+
+SymfonyCloud billing is opt-in based. You will be billed only if you explicitely
+want to keep your project around after trial and accept billing using ``symfony
+project:billing:accept`` or if you move to a Production plan using ``symfony
+project:scale``.
+
+If you do not upscale your project or accept billing, you project will be
+automatically suspended at the end of your trial. If you do no pay in the days
+following its suspension, your project will be automatically deleted.
+
+What is the difference between development and production plans?
+----------------------------------------------------------------
+
+The development plan is intended to let you test and prepare your project before
+it going live. You do not have dedicated resources and can not attach production
+domains to your Main environment.
+
+When a project is attached to a production plan, the Main environment runs on
+different hosts with dedicated ressources and can serve traffic to custom
+production domain names.
+
+How do I change my project plan?
+--------------------------------
+
+To upscale your project one step up (from Standard to Medium for example), use
+``symfony project:scale``. To downscale one step down, use ``symfony
+project:scale --down``.
+
+You can also scale directly to a specific plan using ``symfony project:scale
+<plan>`` (replace ``plan`` with the plan name).
+
+How do I add more storage to my project?
+----------------------------------------
+
+Your subscription comes with 5GB of persistent storage per environment. Adding
+more storage is made per increment of 5GB and is billed $3.60 / 3,00€ per
+increment and per environment.
+
+Add more storage using ``symfony project:edit storage X`` (replacing ``X`` by
+the desired storage amount).
+
+How do I add more environments to my project?
+---------------------------------------------
+
+Your subscription comes with 3 non-production and active environments. You can
+add more environment by increment of 3, billed $25 / 21€ per increment.
+
+Add more environments using ``symfony project:edit environments X`` (replacing
+``X`` by the desired environments quota).
+
+How do I cancel my project?
+---------------------------
+
+You can cancel your SymfonyCloud subscription at any time using ``symfony
+project:delete``. You will be prompted if you want to delete your project
+immediately or automatically at the end of your current billing period.
+
+How can I change my billing details?
+------------------------------------
+
+At the moment it is not possible to edit your billing details yourself. If you
+need to edit them, please file a ticket support at
+https://symfony.com/cloud/support.
```

Comment:
You should add the query string to auto-select SymfonyCloud in the form

Reply:
`/cloud/support` (vs `/support`) already does that

Reply:
Should be a proper link